src/pages/MapPage/components/SearchBar/SearchBar.js

```diff
@@ -128,7 +129,7 @@ const SearchBar = React.forwardRef((props, ref) => {
         )}
         <InputBase
           id='inputBase'
-          inputRef={materialRef}
+          inputRef={showControl === true ? materialRef : ''}
```

Comment:
inputRef's prop type can't be string, please use 'null' instead.

Reply:
OK already change !